packages/manager/modules/hub/src/components/carousel/carousel.controller.js

```diff
@@ -0,0 +1,57 @@
+import map from 'lodash/map';
+import first from 'lodash/first';
+import set from 'lodash/set';
+import find from 'lodash/find';
+import findIndex from 'lodash/findIndex';
+import filter from 'lodash/filter';
+
+import { Environment } from '@ovh-ux/manager-config';
+
+export default class {
+  /* @ngInject */
+  constructor($window, OvhApiNotificationAapi, TranslateService) {
+    this.$window = $window;
+    this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+    this.TranslateService = TranslateService;
+    this.REGION = Environment.getRegion();
```

Comment:
we usually use ```coreConfig.getRegion()```

Reply:
This can cause side effects because it doesn't go through angular hooks and `region` always equals to `EU`. that's why as @frenautvh said we use `coreConfig` instead